FINANCES.md

```diff
@@ -1,20 +1,20 @@
 # Finances
 
-The Tessel Project (as of May 2016) has its funds held by the non-profit Dojo Foundation.
+The Tessel Project (as of May 2016) has its funds held by non-profit foundations. The funds are currently held by the Bocoup Foundation.
```

Comment:
Just the Bocoup Foundation now, though there may still be some disentangling in process? @johnnyman727 can you comment?